azure/pas-single-component-deploy-in-customer-network.json

```diff
@@ -0,0 +1,289 @@
+{
+    "$schema": "http://schema-management-azure-com.o365.apps.cyberark.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "License Agreement": {
+            "defaultValue": "Decline",
+            "allowedValues": [
+                "Accept",
+                "Decline"
+            ],
+            "maxLength": 6,
+            "type": "String",
+            "metadata": {
+                "description": "I have read and agree to the Terms and Conditions."
```

Comment:
I have read and agreed to the Terms and Conditions.

Reply:
Done